app/helpers/office_helper.rb

```diff
@@ -10,6 +12,10 @@ def office_address(office)
   end
 
   def display_website(office, firm)
-    office.website.present? ? office.website : firm.website_address
+    user_uri = URI((office.website.presence || firm.website_address).to_s)
```

Comment:
👍 

Just to add to the existing discussions, _personally_ I think most of the confusion comes from the fact that, originally, `display_website` already did something quite unusual: looking for a website either from an office or a firm – something that usually could be done upfront, by a presenter for instance. Something that [could have been solved upfront in `rad` for instance](https://github.com/moneyadviceservice/rad/blob/3a98660da553354fe22b7c7d1ce1057dd78a47e5/app/serializers/firm_serializer.rb#L6). Something that I've noticed recently when working on decoupling the app from the db, and that will be fixed [if that piece of work will be done eventually](https://github.com/moneyadviceservice/rad/blob/9965-spike-5d-find-alternative-to-elasticsearchpostgres/app/serializers/firm_serializer.rb#L53-L55).

Other than this fallback that was already in place and that could be removed as just explained, the rest of the method LGTM. 👍